src/renderer/remote-loader.ts

```diff
@@ -135,11 +137,29 @@ export class RemoteLoader {
   public async setElectronVersionWithRef(ref: string): Promise<boolean> {
     const version = await this.getPackageVersionFromRef(ref);
 
+    let downloadMissing = false;
     if (!this.appState.hasVersion(version)) {
-      this.handleLoadingFailed(
-        new Error('Version of Electron in example not supported'),
-      );
-      return false;
+      downloadMissing = await this.askToDownloadMissingVersion(version);
```

Comment:
... does it make sense to prompt the user before downloading a version of Electron? We download them pretty freely everywhere else in the app, so I'm wondering why the app should handle it differently here.

Reply:
That's a good point and i'm also generally happier to take the path of least resistance especially when it's already being taken elsewhere!